packages/ui/src/views/repo/repo-list/repo-list-page.tsx

```diff
@@ -14,7 +15,16 @@ import { sortRepositories } from '../utils/sorting/repos'
 import { RepoList } from './repo-list'
 import { RepoListProps } from './types'
 
-const LinkComponent = ({ to, children }: { to: string; children: ReactNode }) => <Link to={to}>{children}</Link>
+const LinkComponent = ({ to, children, disabled }: { to: string; children: ReactNode; disabled?: boolean }) => (
```

Comment:
why is this needed?
if needed, LinkComponent should be defined in the app layer and passed to the view.
View should not be defining a LinkComponent

Reply:
yes not required, probably transferred from old views of playground, removing this